### ISSUE TITLE
Allow wildcards in fallbacks entries

### ISSUE THREAD

**Issue by adriantanasa** [mashpie/i18n-node#361]
This is a feature request to allow wildcard locales in the fallbacks ex:

```javascript
// ex: fallback for all de-AT / de-CH / de-DE / de-LI / de-LU to de
i18n.configure({
        locales: ['en', 'de'],
        defaultLocale: 'en',
        fallbacks: { 'de-*': 'de' },
        directory: './locales',
        register: req
      });
```
Using the current implementation entire list needs to be provided in fallbacks

**Comment by gzurbach** [comment]
I was honestly expecting this to be the default behavior. I spent the last hour wondering why my `fr-FR,en-US;q=0.7,en;q=0.3`header was being identified as `en` with the following config:

```
  // Configure i18n
  i18n.configure({
    locales: ['en', 'fr'],
    defaultLocale: 'en'
  });
```

@adriantanasa Have you found any workaround other than listing every possible french locale in the `fallbacks` config?

**Comment by adriantanasa** [comment]
@gzurbach No. Just listing all the locales as you did.

**Comment by mashpie** [comment]
0.11.0 released to npm with fixing PR #362
